pywinauto/unittests/test_UIAWrapper.py

```diff
@@ -666,20 +670,27 @@ def test_get_header_control(self):
 
         def test_select(self):
             """Test selecting an item of the ListView control"""
+            # Verify get_selected_count
+            cnt = self.ctrl.get_selected_count()
+            self.assertEqual(cnt, 0)
```

Comment:
I think it should be fine as a one liner: `self.assertEqual(self.ctrl.get_selected_count(), 0)`